templates/pages/tools/project_task.html.twig

```diff
@@ -0,0 +1,289 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% extends "generic_show_form.html.twig" %}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% set params  = params ?? [] %}
+
+{% block form_fields %}
+
+    {{ fields.dropdownField(
+        'ProjectTaskTemplate',
+        'projecttasktemplates_id',
+        item.fields.projecttasktemplates_id,
+        _n('Project task template', 'Project task templates', 1),
+        {
+            'entity': item.getEntityID(),
+            'on_change': 'projecttasktemplate_update(this.value)'
+        }
+    ) }}
+
+    <script>
+        function projecttasktemplate_update(value) {
+            $.ajax({
+                url: CFG_GLPI.root_doc + "/ajax/projecttask.php",
+                type: "POST",
+                data: {
+                    projecttasktemplates_id: value
+                }
+            }).done(function(data) {
+                // Set simple inputs
+                $('input[name=name]').val(data.name);
+                $('textarea[name=comment]').val(data.comments);
```

Comment:
Selectors are not precise enough, `name` and `comment` are common field names. You should either find a way to trigger the form.
Maybe something like:
```js
function projecttasktemplate_update(value) {
    const form = $(this).closest('form');
```
and then:
```js
form.find('input[name=name]').val(data.name);
```